internal/v4/content_test.go

```diff
@@ -20,7 +20,7 @@ import (
 	"gopkg.in/juju/charmrepo.v2-unstable/csclient/params"
 
 	"gopkg.in/juju/charmstore.v5-unstable/internal/storetesting"
-	"gopkg.in/juju/charmstore.v5-unstable/internal/v4"
+	"gopkg.in/juju/charmstore.v5-unstable/internal/v5"
```

Comment:
Perhaps not so weird if you consider that all the API versions are implemented by reference to the latest version, so that the latest version is the cleanest and old versions can eventually be removed easily.

It's useful in this case because most of the v4 tests are identical (copy/pasted) from v5, so having a v4 vs v5 difference in the code can make the difference between a patch succeeding or failing.